ccmlib/common.py

```diff
@@ -399,20 +399,25 @@ def isOpscenter(install_dir):
     opscenter_script = os.path.join(bin_dir, 'opscenter')
     return os.path.exists(opscenter_script)
 
+def scylla_extract_mode(path):
+    m = re.match('(.*/)?build/(\w+)(/|$)', path)
```

Comment:
can you add a few examples here ? i.e. one url one file path ?

Reply:
Added in v2